src/main/java/org/spongepowered/common/data/builder/block/tileentity/SpongeBrewingStandBuilder.java

```diff
@@ -48,24 +48,24 @@ public SpongeBrewingStandBuilder(Game game) {
     public Optional<BrewingStand> build(DataView container) throws InvalidDataException {
```

Comment:
As per @gabizou's comments, you can leave this alone - the builder system is going to be refactored.